packages/js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@novu/js",
+  "version": "0.24.2",
+  "repository": "https://github.com/novuhq/novu",
+  "description": "Novu's JavaScript SDK for building custom inbox notification experiences",
+  "author": "",
+  "license": "ISC",
+  "main": "dist/index.js",
```

Comment:
I'd suggest using conditional exports instead of `main`, `module`, `type`. They provide more control, and they have become the [new defacto standard](https://stackoverflow.com/questions/68572936/what-is-the-difference-between-main-and-module-vs-exports-in-package-json). 

The assumption is that we will also include them to re-export the JS UI part.